src/w3c/templates/headers.js

```diff
@@ -2,16 +2,42 @@ import "deps/hyperhtml";
 import showLogo from "./show-logo";
 import showPeople from "./show-people";
 import showLink from "./show-link";
+import { pub } from "core/pubsubhub";
 
 export default conf => {
   const html = hyperHTML;
+  const specTitleElem =
+    document.querySelector("h1#title") || document.createElement("h1");
+  if (specTitleElem.parentElement) {
+    specTitleElem.remove();
+    conf.title = specTitleElem.textContent.trim();
+  } else {
+    if (document.title !== conf.title) {
+      pub("warn", "The document's title and the `<title>` element differ.");
+    }
+    specTitleElem.textContent = conf.title;
+    specTitleElem.id = "title";
+  }
+  specTitleElem.classList.add("title", "p-name");
+
+  let specSubTitleElem = null;
+  if (conf.subtitle) {
```

Comment:
@shikhar-scs As you are using condition `if (conf.subtitle)`, if `conf.subtitle` is not set, then even if a `<h2 id="subtitle">SUB</h2>` is present, it won't be used (i.e. the subtitle won't appear in `.head`).
Please check. (https://i.imgur.com/AO1cmSG.png)

Reply:
Ohk. I'll see this in a while :+1:

Reply:
@sidvishnoi nice find. Definitely need add that as a test!

Reply:
In such cases should I then introduce `conf.subtitle` in the configs, based on the condition `if(!conf.subtitle && document.querySelector("h2#subtitle")` ? 
Or just keep a test to ensure that all of this is in place ?

Reply:
Both, I would say. But basically, the tests should guide the code. Just add sufficient tests to cover all possible situations. If nothing breaks, you are good :)

Reply:
Ok :+1: